cognee/infrastructure/databases/vector/weaviate_db/WeaviateAdapter.py

```
@@ -172,28 +172,33 @@ async def get_distance_from_collection_elements(
-            for result in search_result.objects
-        ]
+
+            return [
+                ScoredResult(
+                    id=parse_id(str(result.uuid)),
+                    payload=result.properties,
+                    score=1 - float(result.metadata.score),
+                )
+                for result in search_result.objects
+            ]
+        except weaviate.exceptions.UnexpectedStatusCodeError as error:
```

Ruff (0.8.2)

Local variable `error` is assigned to but never used

Remove assignment to unused variable `error`

(F841)